app/views/verify/address/index.html.slim

```diff
@@ -3,13 +3,27 @@ h1.h2
 p
   = decorated_session.verification_method_choice
 
-.clearfix
-  .sm-col.sm-col-12.md-col-6
-    = link_to t('idv.buttons.activate_by_phone'), verify_phone_path,
-      class: 'btn btn-primary mb2 center inline-block'
+= form_tag
+  label.block.p2.border.rounded-md.border-light-blue.mb2 for="address_delivery_method_phone"
+    .radio.mb2
+      = radio_button_tag 'address_delivery_method', :phone
+      span.indicator
+      strong.blue.block = t('idv.messages.select_verification_form.phone_header')
+      span = t('idv.messages.select_verification_form.phone_message')
+  
+  // USPS verification should always be enabled in prod.
+  // This check is a convenience for lower envs.
   - if FeatureManagement.enable_usps_verification?
-    .sm-col.sm-col-12.md-col-6
-      = link_to t('idv.buttons.activate_by_mail'), verify_usps_path,
-        class: 'btn btn-outline rounded-lg mb2 center'
+    label.block.p2.border.rounded-md.border-light-blue.mb2 for="address_delivery_method_usps"
+      .radio.mb2
+        = radio_button_tag 'address_delivery_method', :usps
+        span.indicator
+        strong.blue.block = t('idv.messages.select_verification_form.usps_header')
+        span = t('idv.messages.select_verification_form.usps_message')
 
+  = button_tag t('forms.buttons.continue'), type: 'submit',
+    class: 'btn btn-primary col-6 mb2 p2 rounded-lg inline-block'
+
+// Be sure to keep this outdented to the same level as the form above
+// or terrible things will happen to your tests.
```

Comment:
Same here. We can probs drop this comment

Reply:
I'm on the fence on this one. On the one hand, I tend to like to comment anyplace a simple mistake can lead to big headaches. On the other hand, this one was really just my silly mistake, and I can see an argument that the comment only says "malformed HTML is bad". So I could go either way.